src/khoj/routers/helpers.py

```diff
@@ -111,7 +111,7 @@ def _save_to_conversation_log(
                 )
             else:
                 loaded_model = state.processor_config.conversation.gpt4all_model.loaded_model
-                chat_response = converse_falcon(
+                chat_response = converse_llama(
```

Comment:
should we just call this converse_offline instead of changing it to name of every new offline chat model we may migrate to?
```suggestion
                chat_response = converse_offline(
```